override/__init__.py

```diff
@@ -0,0 +1,6 @@
+from .override import *
+
+
+def setup(bot):
+    pdb = Override(bot)
```

Comment:
Override seems like a bad name for a cog. Can you change it to SelfRoleOverride, or CustomSelfRole, or SelfRole2.